imagevector/images.yaml

```diff
@@ -1,5 +1,5 @@
 images:
 - name: dns-controller-manager
   sourceRepository: github.com/gardener/external-dns-management
-  repository: eu.gcr.io/gardener-project/dns-controller-manager
+  repository: europe-docker.pkg.dev/gardener-project/releases/dns-controller-manager
```

Comment:
The project https://github.com/gardener/external-dns-management has not yet switched to the artifact registry. 
Shouldn't it switched before we change the repository here?

Reply:
@MartinWeindel : that depends on how optimistic we are considering the time of switching external-dns-manager. I already replicated (manually, as a one-off operation) all referenced OCI Images. So for the status quo, changing reference to AR will not result in a dangling reference. It is also a valid approach to wait and do those changes along "dependency-tree".

Reply:
@ccwienk 
I would prefer to wait at least until the switch PR for the external-dns-management project is merged. If there is a need for an urgent release of the dns-controller-manager, it would currently only be available at `eu.gcr.io` and releasing shoot-dns-service would not be possible without reverting some changes.

Reply:
sure, sounds reasonable. there is no need to rush those changes.